plugins/inputs/kinesis_consumer/kinesis_consumer.go

```diff
@@ -118,6 +131,15 @@ var sampleConfig = `
   ## https://github.com/influxdata/telegraf/blob/master/docs/DATA_FORMATS_INPUT.md
   data_format = "influx"
 
+  ##
+  ## Whether or not to uncompress the data from kinesis
+  ## If you are processing a cloudwatch logs kinesis stream then set this to "gzip"
+  ## as AWS compresses cloudwatch log data before it is sent to kinesis (aws
+  ## also base64 encodes the zip byte data before pushing to the stream.  The base64 decoding
+  ## is done automatically by the golang sdk, as data is read from kinesis)
+  ##
+  # decompress_with = "none"
```

Comment:
How about making `""` a synonym for `none`? This way you do get the default for free. :-)